cmd/cheqd-noded/cmd/root.go

```diff
@@ -190,14 +191,17 @@ func (a appCreator) newApp(logger log.Logger, db dbm.DB, traceStore io.Writer, a
 	}
 
 	snapshotDir := filepath.Join(cast.ToString(appOpts.Get(flags.FlagHome)), "data", "snapshots")
-	snapshotDB, err := sdk.NewLevelDB("metadata", snapshotDir)
+	snapshotDB, err := dbm.NewDB("metadata", dbm.CLevelDBBackend, snapshotDir)
```

Comment:
This used to return `goleveldb` as a flag.
Switched to a more literal one, since we can compile for `goleveldb`, `gorocksdb`, given our dependencies in `go.mod`:
```go
snapshotDB, err := dbm.NewDB("metadata", dbm.GoLevelDBBackend, snapshotDir)
```